app/controllers/api/v0/api_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Api::V0::ApiController < ApplicationController
   include PagingConcern
-  before_action :require_authentication!, only: [:create]
+  before_action :require_authentication!
```

Comment:
While we’re here, I think it would be good to adjust the `require_authentication!` method to actually raise an `Api::AuthorizationError` (you’d need to create that in `lib/api/api.rb`) or `Api::DynamicError`. I should have written it that way from the beginning, sorry :\

Reply:
No problem! Done, covered with a test, force-pushed.